### PR TITLE
Better obsoletes for ISatellite

### PR DESCRIPTION
**Pull request by andreasohlund**
Now points to AddSatelliteReceiver

cc @mikeminutillo

### CODE REVIEW

File: src/NServiceBus.Core/obsoletes.cs [hunk 1]
```diff
@@ -1765,15 +1765,15 @@ public void Replace(WellKnownStep wellKnownStep, Type newBehavior, string descri
 namespace NServiceBus.Satellites
 {
     [ObsoleteEx(
-        Message = "ISatellite is no longer an extension point. In order to create a satellite one must create a feature that uses AddSatelliteReceiver() method and a class that inherits from PipelineTerminator<ISatelliteProcessingContext> that is used for processing the messages.",
+        Message = "No longer an extension point. Instead create a feature and use FeatureSetupContex.AddSatelliteReceiver(...).",
```
Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: `FeatureSetupContext`

File: src/NServiceBus.Core/obsoletes.cs [hunk 2]
```diff
@@ -1765,15 +1765,15 @@ public void Replace(WellKnownStep wellKnownStep, Type newBehavior, string descri
...
         RemoveInVersion = "7",
         TreatAsErrorFromVersion = "6")]
     public interface IAdvancedSatellite
     {
     }
 
     [ObsoleteEx(
-        Message = "ISatellite is no longer an extension point. In order to create a satellite one must create a feature that uses AddSatelliteReceiver() method and a class that inherits from PipelineTerminator<ISatelliteProcessingContext> that is used for processing the messages.",
+        Message = "No longer an extension point. Instead create a feature and use FeatureSetupContex.AddSatelliteReceiver(...).",
```
Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: `FeatureSetupContext`

File: src/NServiceBus.Core/obsoletes.cs (hunk 1 above)
Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: Should `feature` refer to the class Feature?

File: src/NServiceBus.Core/obsoletes.cs [hunk 3]
```diff
@@ -1765,15 +1765,15 @@ public void Replace(WellKnownStep wellKnownStep, Type newBehavior, string descri
 namespace NServiceBus.Satellites
 {
     [ObsoleteEx(
-        Message = "No longer an extension point. Instead create a feature and use FeatureSetupContex.AddSatelliteReceiver(...).",
+        Message = "No longer an extension point. Instead create a feature and use FeatureSetupContext.AddSatelliteReceiver(...).",
```
Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: Sorry it should be `FeatureConfigurationContext `

**Comment by danielmarbach** [comment]
@andreasohlund I got trapped by you ;) It should be `FeatureConfigurationContext `

**Comment by andreasohlund** [comment]
Should be ok now

File: src/NServiceBus.Core/obsoletes.cs (hunk 1 above)
Comment (andreasohlund) [inline src/NServiceBus.Core/obsoletes.cs]: Is that supported in obsoletes? (talking about a `<ref ... />` ?)

Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: Thought about capitalizing feature. No ref

Comment (andreasohlund) [inline src/NServiceBus.Core/obsoletes.cs]: Fixed, :shipit: :)

**Pull request merged**